### PR TITLE
Created user guide and modified main doc

### PR DESCRIPTION
**Pull request by yuriatgoogle**
Created user guide in /docs
Added images for user guide
added link to user guide on main doc under "next steps"

### CODE REVIEW

**Comment by yuriatgoogle** [comment]
@simonz130 I don't know why the build is failing - these are just doc changes.

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+
+The installation process takes a few minutes. When it completes, you see a message like the following:
+
+```bash
+********************************************************************************
+Stackdriver Sandbox deployed successfully!
+
+     Stackdriver Dashboard: https://app.google.stackdriver.com/accounts/create
+     Google Cloud Console Dashboard: https://console.cloud.google.com/kubernetes/workload?project=stackdriver-sandbox-68291054
+     Hipstershop web app address: http://35.202.126.83
```
Comment (simonz130) [inline docs/README.md]: please mask the IP address, like http://A.B.C.D

**Comment by simonz130** [comment]
Please sync with master

**Comment by yuriatgoogle** [comment]
@simonz130 updated readme to hide IP addresses.  Ready to merge?

**Comment by yuriatgoogle** [comment]
@simonz130 just following on this - what's holding this up?

**Pull request merged**